Alignment/APEEstimation/test/cfgTemplateDesign/createStep1.bash

```diff
@@ -42,12 +42,12 @@ export datasetName
 ## number of input files
 export nFiles
 ## Input file base
-cafDir="\/store\/caf\/user\/cschomak\/wlnu"
+cafDir="\/store\/caf\/user\/cschomak\/Skims\/MC\/PhaseI\/DY"
```

Comment:
I recommend to replace your user name with `$USER` or is this a fixed location also used by other users?